takhojin/app.js

```diff
@@ -0,0 +1,40 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+dotenv.config();
+
+const myDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized");
+});
+
+app = express();
```

Comment:
```suggestion
const app = express();
```
변수가 수정되지 않도록 const를 활용해주세요!